HWIMO-BUILD

```diff
@@ -9,6 +9,12 @@
 set -e
 set -x
 
+# $1 should be a file which contains the release version.
+# For example:  PKG_VERSION=on-http~1.2.3-rc-abc...
+if [ -f "$1" ];then
```

Comment:
Out of curiosity, why pass the version in a file instead of pass it as a command line argument?